internal/config/storage.go

```diff
@@ -60,7 +62,15 @@ func (c *StorageConfig) setDefaults(v *viper.Viper) error {
 			v.SetDefault("storage.object.s3.poll_interval", "1m")
 		}
 	case string(OCIStorageType):
```

Comment:
do we need to update the JSON and cue schemas as well for this?

Reply:
Ah yes, thank you!